docs/ADD_NEW_EXCHANGE.md

```diff
@@ -1053,32 +1161,39 @@ Below are the features supported by FTX API protocol:
 Initially the functions return nil or common.ErrNotYetImplemented
 
 ```go
-// GetWebsocket returns a pointer to the exchange websocket
-func (f *FTX) GetWebsocket() (*wshandler.Websocket, error) {
-	return f.Websocket, nil
+// AuthenticateWebsocket sends an authentication message to the websocket
+func (f *FTX) AuthenticateWebsocket() error {
+	return f.WsAuth()
 }
+```
 
-// SubscribeToWebsocketChannels appends to ChannelsToSubscribe
-// which lets websocket.manageSubscriptions handle subscribing
-func (f *FTX) SubscribeToWebsocketChannels(channels []wshandler.WebsocketChannelSubscription) error {
-	f.Websocket.SubscribeToChannels(channels)
-	return nil
-}
 
-// UnsubscribeToWebsocketChannels removes from ChannelsToSubscribe
-// which lets websocket.manageSubscriptions handle unsubscribing
-func (f *FTX) UnsubscribeToWebsocketChannels(channels []wshandler.WebsocketChannelSubscription) error {
-	f.Websocket.RemoveSubscribedChannels(channels)
-	return nil
-}
+## Last but not least - Live testing
 
-// GetSubscriptions returns a copied list of subscriptions
-func (f *FTX) GetSubscriptions() ([]wshandler.WebsocketChannelSubscription, error) {
-	return f.Websocket.GetSubscriptions(), nil
-}
+### Live testing websocket via [gctcli](../cmd/gctcli/main.go)
 
-// AuthenticateWebsocket sends an authentication message to the websocket
-func (f *FTX) AuthenticateWebsocket() error {
-	return f.WsAuth()
-}
-```
\ No newline at end of file
+- Run all websocket command below while a gocryptotrader instance is running and the exchange websocket is enabled; ensure fetching of information is possible, disable/enable websocket disconnects and reconnects connection, ensure subscriptions are valid, ensure proxy can be set and reset and ensure URL can be set in the event that there might be an api endpoint change while a persistant instance is up and running.  
```

Comment:
Please test all websocket commands below whilst a GoCryptoTrader instance is running and with the exchange websocket setting enabled:

- `getinfo` to ensure fetching websocket information is possible (that the websocket connection is enabled, connected and is running.
- `disable/enable` to ensure disabling/enabling a websocket connection disconnects/connects accordingly.
- `getsubs` to ensure the subscriptions are in sync with the exchange's config settings or by manual subscriptions added/removed via `gctcli`.
- `setproxy` to ensure that a proxy can be set and resets the websocket connection accordingly.
- `seturl` to ensure that a new websocket URL can be set in the event of an API endpoint change whilst an instance of GoCryptoTrader is already running.